client/state/comments/actions.js

```diff
@@ -56,21 +56,20 @@ export const requestCommentsList = query => ( {
 	query,
 } );
 
-/***
- * Creates a remove comment action for a siteId, postId, commentId
+/**
+ * Creates an action that permanently deletes a comment
```

Comment:
What happens if we delete a comment that doesn't have the current status of `trash`?